### PR TITLE
Ports the UDP Client from Aurora RusT-G

### PR DESCRIPTION
**Pull request by Iamgoofball**
Ports https://github.com/Aurorastation/rust-g/pull/5/

### CODE REVIEW

**Comment by Cyberboss** [comment]
What for?

**Comment by optimumtact** [comment]
your haptic feedback plugin is never going to get merged to tg mainline because we're not in the business of adding features only 1 person can use.

I don't see a problem with adding this to rust-g itself though, although it doesn't seem very useful

File: Cargo.toml [hunk 1]
```diff
@@ -41,7 +41,7 @@ rand = {version = "0.7", optional = true}
 
 
 [features]
-default = ["cellularnoise", "dmi", "file", "git", "http", "json", "log", "noise", "sql"]
+default = ["cellularnoise", "dmi", "file", "git", "http", "json", "log", "noise", "sql", "udp_shipper"]
```
Comment (ZeWaka) [inline Cargo.toml:44]: lmao no

File: src/lib.rs [hunk 2]
```diff
@@ -30,6 +30,7 @@ pub mod url;
 pub mod unzip;
 #[cfg(feature = "cellularnoise")]
 pub mod cellularnoise;
-
+#[cfg(feature="udp_shipper")]
+pub mod udp_shipper;
```
Comment (ZeWaka) [inline src/lib.rs:34]:
```suggestion
pub mod udp_shipper;

```

**Review by ZeWaka — CHANGES_REQUESTED** [review]
no entry in dmsrc as well

**Comment by AffectedArc07** [comment]
Feeling like closing this given it hasn't been touched in over 10 months,

**Comment by Iamgoofball** [comment]
i forgot it was still open myself

**Pull request closed without merging**